services/secondary-storage/controller/src/main/java/org/apache/cloudstack/secondarystorage/SecondaryStorageManagerImpl.java

```diff
@@ -382,15 +347,16 @@ public boolean generateVMSetupCommand(Long ssAHostId) {
         if (ssAHost.getType() != Host.Type.SecondaryStorageVM) {
             return false;
         }
-        SecondaryStorageVmVO secStorageVm = _secStorageVmDao.findByInstanceName(ssAHost.getName());
+        String hostName = ssAHost.getName();
+        SecondaryStorageVmVO secStorageVm = _secStorageVmDao.findByInstanceName(hostName);
         if (secStorageVm == null) {
-            s_logger.warn("secondary storage VM " + ssAHost.getName() + " doesn't exist");
+            s_logger.warn(String.format("Secondary storage VM [%s] does not exist.", hostName));
```

Comment:
I would rename the variable from `hostName` to something that fits better on the method context.
Example: `ssvmName`.

Reply:
@GabrielBrascher done.

Reply:
Thanks @GutoVeronezi!